packages/fluentui/react-northstar/src/themes/teams/components/Tree/treeTitleVariables.ts

```diff
@@ -1,13 +1,52 @@
 import { pxToRem } from '../../../../utils';
+import * as _ from 'lodash';
 
 export interface TreeTitleVariables {
   color: string;
+  indicatorColor: string;
+
   padding: string;
+  gap: string;
```

Comment:
I don't see any usage this variable

Reply:
fixed, removed :) it was caused by copy from checkbox